src/plugins/intel_gpu/tests/functional/subgraph_tests/dynamic/kv_cache.cpp

```diff
@@ -99,6 +99,19 @@ const std::vector<std::vector<InputShape>> input_shapes_basic = {
     },
 };
 
+void resize_function(std::shared_ptr<ov::Model> function, const std::vector<ov::Shape>& targetInputStaticShapes) {
+    auto inputs = function->inputs();
+    std::map<ov::Output<ov::Node>, ov::PartialShape> shapes;
+    if (inputs.size() > targetInputStaticShapes.size()) {
```

Comment:
As this is test code use gtest assertion ASSERT_GT() << "error message"; ?

Reply:
This is a free function not related to direct code execution. So I would not use gtest macroces.
But there is no reason to use std::runtime_error. So I changed it to OPENVINO_ASSERT